app/org/maproulette/framework/model/LockedTask.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2020 MapRoulette contributors (see CONTRIBUTORS.md).
+ * Licensed under the Apache License, Version 2.0 (see LICENSE).
+ */
+package org.maproulette.framework.model
+
+import org.joda.time.DateTime
+import play.api.libs.json.{Json, Format}
+import play.api.libs.json.JodaWrites._
+import play.api.libs.json.JodaReads._
+
+/**
+  * Mapping of object structure for fetching task lock data
+  */
+case class LockedTaskData(
+    id: Long,
+    parent: Long,
+    parentName: Option[String],
+    startedAt: DateTime
```

Comment:
Please add javadoc explaining what this field indicates. I don't want to require a better name than `startedAt`, but it should be clarified at the very least with javadoc.
Like
```
/**
  * The time that the task was locked
  */
```

Reply:
Kinda close, the fields themselves is what the javadoc goes on, like

```
/**
  * The time that the task was locked
  */
    startedAt: DateTime
```